src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const ofSArr = sourceString.split(';');
+  const opArray = ofSArr.map(x => x.trim().split(':').map(y => y.trim()));
```

Comment:
opArray bad name, don't use types as a name.
x, y => bad names, don't use letters as a name.
no need to use trim on every step, use it before adding it into an object.